### DISCUSSION TITLE
Don't work aright.

### DISCUSSION THREAD

**Discussion by Azir233** [hoothin/UserScripts#436] [Q&A]
The default api --（[Lolicon ACG SexyPic][Nyan ACG SexyPic][3650000]）can not work in my edge.
API calls normal,but can't display pictures properly.
This is a screenshot for console:
Please help me.
![图片1](https://github.com/hoothin/UserScripts/assets/141922790/e370a796-ac4e-46ac-8a20-987e6edf69ae)

**Comment by hoothin** [comment]
If you are using Edge, make sure JSON Viewer in edge://flags/ is disabled.

你这。。。我只是想要占个坑避免被人抄袭才传到油叉上去的，这就是个私人用途的脚本。我没推广也没打算分享给别人用，甚至中文介绍都删掉了，怎么还是总有人来提要求？

**Reply by Azir233** [reply]
原来是说中文的啊😂

**Reply by Azir233** [reply]
就想请教一下，前几个版本还能用，最近用不了。

**Reply by hoothin** [reply]
上面不是说了吗？JSON Viewer in edge://flags/

**Reply by Azir233** [reply]
谢谢大哥

**Reply by hoothin** [reply]
JSON Viewer相当于垃圾浏览器edge的内置插件，一旦开启，打开json源码会自动替换成高亮美化界面。

**Reply by Azir233** [reply]
明白了，谢谢
